Sources/Private/CoreAnimation/Animations/LayerProperty.swift

```diff
@@ -390,6 +391,15 @@ extension CustomizableProperty {
       })
   }
 
+  static var path: CustomizableProperty<CGPath> {
```

Comment:
How have you been testing this? Can you add a snapshot test case for this functionality? 

You'll need to add a case for this to `SnapshotConfiguration.swift`, you can follow the examples of the existing value provider test cases. If you add a new sample animation file, you can add it at `Tests/Samples/Issues/pr_2376.json`.

Reply:
I encountered an issue with the tests, but I think it may be a problem with how the tests work.

`recordHierarchyKeypath`, which is used to log all the keypaths in use by the Core Animation engine, logs whatever is used in a ValueProvider. This now includes a `Path` property.

The test fails because the `Path` property seems to be missing in the Main Thread engine.

But the PathValueProvider works for both, so I think the test is at fault.

I will add Snapshot tests as you suggested.